public/js/index.js

```diff
@@ -177,7 +194,9 @@ $(function () {
     success: function (data) {
       $.each(data.widgets, function (i, widget) {
         createWidget(widget, objSocket)
+        
```

Comment:
Nit: This blank line can be removed, too.

Reply:
done, I'll try removing these when I notice them